### PR TITLE
Run the code inspector

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

File: news-bundle/src/Resources/contao/modules/ModuleNewsReader.php [hunk 1]
```diff
@@ -16,13 +16,15 @@
...
- * Front end module "news reader".
+ * Front end module "newsreader".
  *
  * @property Comments $Comments
  * @property string   $com_template
  * @property array    $news_archives
  *
  * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @todo Rename to ModuleNewsreader in Contao 5.0
```
Comment (fritzmg) [inline news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: Why? Then we also need to rename `ModuleNewsList` to `ModuleNewslist`, don't we?

**Review by ausi — APPROVED** [review]

File: news-bundle/src/Resources/contao/modules/ModuleNewsReader.php (hunk 1 above)
Comment (leofeyer) [reply news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: No, because "newslist" is not a word but "newsreader" is. 🤷‍♂️

Comment (leofeyer) [reply news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: But that also seems debatable.

Comment (leofeyer) [reply news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: It seems that "newsreader" means "Nachrichtensprecher", so we should actually stick with "news reader". 🤦‍♂️ I am going to adjust the code inspector accordingly.

Comment (leofeyer) [reply news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: Reverted in f6a1068b74b819a2874aa9f74f25b4d2169b92ed.

Comment (fritzmg) [reply news-bundle/src/Resources/contao/modules/ModuleNewsReader.php:27]: I see - yes that should be changed.

**Pull request merged**